File: 0014-longest-common-prefix/0014-longest-common-prefix.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        answer = ""
        
        strs = sorted(strs)
        start = strs[0]
        end = strs[-1]
        for i in range(min(len(start),len(end))):
            if(start[i]!=end[i]):
                return answer
            answer+=start[i]
        
        return answer
        

File: 0014-longest-common-prefix/0014-longest-common-prefix.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        String answer = "";
        Arrays.sort(strs);
        String start = strs[0];
        String end = strs[strs.length-1];
        int size = Math.min(start.length(), end.length());
        for ( int i =0 ; i < size; i++){
            if(start.charAt(i) != end.charAt(i)){
                return answer;
            }
            answer += start.charAt(i);
        }
        
        return answer;
    }
}